addons/point_of_sale/models/pos_order.py

```diff
@@ -348,10 +352,10 @@ def _default_pricelist(self):
     company_id = fields.Many2one('res.company', string='Company', required=True, readonly=True, default=lambda self: self.env.user.company_id)
     date_order = fields.Datetime(string='Order Date', readonly=True, index=True, default=fields.Datetime.now)
     user_id = fields.Many2one('res.users', string='Salesman', help="Person who uses the cash register. It can be a reliever, a student or an interim employee.", default=lambda self: self.env.uid)
-    amount_tax = fields.Float(compute='_compute_amount_all', string='Taxes', digits=0)
-    amount_total = fields.Float(compute='_compute_amount_all', string='Total', digits=0)
-    amount_paid = fields.Float(compute='_compute_amount_all', string='Paid', states={'draft': [('readonly', False)]}, readonly=True, digits=0)
-    amount_return = fields.Float(compute='_compute_amount_all', string='Returned', digits=0)
```

Comment:
@legalsylvain 
For the simple reason that in Austria we have very strict regulations how a cash transaction must be handled where the PoS is the place to be and document normally. Explicitly, we need to register all transactions by VAT type, accumulate the brut amount and encrypt and chain the transaction with the predecessor and print it on the ticket as QR Code.
If we make it possible to create or even change orders in the backend it is a high risk of breaking the whole process.

But yeah, it is always an option to do this during customization. IMHO if you want to do stuff outside of the PoS frontend, you would go for sale order and invoice and it is still possible to reference on a cash transaction in the PoS to an invoice for example.

Anyway, I vote for keeping all PoS transactions related actions to ```pos.order``` in the frontend to not mix the process and therefore increase the complexity.